Sources/PayPalDataCollector/Public/PayPalDataCollector/PPDataCollector.swift

```diff
@@ -15,21 +17,45 @@ import Security
      - Parameter pairingID: A pairing ID to associate with this clientMetadataID must be 10-32 chars long or null
      - Returns: A client metadata ID to send as a header
      */
+    // NEXT_MAJOR_VERSION: make this not a class function
```

Comment:
class/static functions are a bit weird in this file, guessing it's needed because of how we call into this class is BTDataCollector under the hood? This would also be a breaking change but is weird to test and generally not the best practice, so lots of next major version notes to find a better way to do this in next major version.